firmware/repository.py

```diff
@@ -0,0 +1,126 @@
+"""Repository - module to communicate with the database"""
+
+from firmware.database import db_session
+from firmware.models import User, Company, Category, Review
+from sqlalchemy import and_
+
+
+def get_companies():
+    """ gets all companies from database """
+    companies = db_session.query(Company)
+    return companies
+
+
+def get_user_id(username):
```

Comment:
Please rewrite this method so that it returns a user object. Then you can use any field you want from the user object in the code which called this method.